repos/keg-components/src/components/drawer/drawer.js

```diff
@@ -104,8 +101,10 @@ export const Drawer = props => {
 }
 
 Drawer.propTypes = {
-  initial: PropTypes.number,
+  config: PropTypes.object,
   Element: PropTypes.oneOfType([ PropTypes.func, PropTypes.elementType ]),
+  initial: PropTypes.number,
   styles: PropTypes.oneOfType([ PropTypes.object, PropTypes.array ]),
   toggled: PropTypes.bool,
+  type: PropTypes.string,
 }
```

Comment:
* the `stories.mdx` is super clean and i'm super excited to use it more down the road.
looking at it in the storybook, it would be helpful if we can somehow show the default values here (if they have one)
    - ![image](https://user-images.githubusercontent.com/3317835/94493953-d6806300-01a2-11eb-889e-26364c9ee82b.png)

it seems like we can do so by setting `Drawer.defaultProps`
```
Drawer.defaultProps = {
  type: 'timing',
  ...otherProps
}
```

* another suggestion is to update the `propTypes` for `type` to be something like `  type: PropTypes.oneOf(['timing', 'something else']),` - that way in the docs we can see clearly what values will really make it work

which then will automatically show up in the storybook
![image](https://user-images.githubusercontent.com/3317835/94494357-d7fe5b00-01a3-11eb-9e7c-92f18b7ea7b7.png)

Reply:
Default values are longer going to be a [thing](https://github.com/facebook/react/pull/16210)
No point in adding them if they are going away. May be there's a way to hack it just for the Storyboard
  * I'll look and see

Reply:
So I added them to the component  in the Drawer.example.js file.
  * This way they can show up in the props table, while at the same time not affect the original component

Reply:
nice. that works for me